scan_build.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
```

Comment:
Is it worth a separate script (`scan_build.sh`)? What about incorporating the stuff into `meta_builds.sh`?

Reply:
nope, I just re-used it as it mostly worked OOTB - I'm going to incorporate it.